### PR TITLE
Test Task/Batch workloads on Skipper mode

### LINKED ISSUE #453: Test Task/Batch workloads on Skipper mode
As a follow-up to spring-cloud/spring-cloud-dataflow#2548, we would have to verify all the classic mode functionality in CF-server. While we are at it, we might have to remove extra config properties.

child of spring-cloud/spring-cloud-dataflow#2548

### PR DESCRIPTION
**Pull request by tzolov**
- Remove classic-mode occurrences in the documentation and use the Skipper as a default and only Stream management option
 - Add skipper Uri to all SCDF configurations

Resolves #453

### CODE REVIEW

File: spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/configuration.adoc [hunk 1]
```diff
@@ -632,35 +624,57 @@ applications:
...
+
 ----
 
 where `my-config-server` is the name of the Spring Cloud Config Service instance running on Cloud Foundry.
 ====
 
-By binding the
-service to both Spring Cloud Data Flow server and all the Spring Cloud Stream and Spring Cloud Task applications
-respectively, we can now resolve centralized properties backed by this service.
+By binding the service to both Spring Cloud Data Flow server and Spring Cloud Task and via Skipper to all the Spring Cloud Stream applications respectively, we can now resolve centralized properties backed by this service.
```
[the remark this answers is not in the document]
Comment (tzolov) [reply spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/configuration.adoc]: +1

File: spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc [hunk 2]
```diff
@@ -4,23 +4,19 @@
 [[getting-started-requirements]]
 == System Requirements
 
-The Spring Cloud Data Flow server deploys streams (long-lived applications) and tasks (short-lived applications) to Cloud
-Foundry. The server is a lightweight Spring Boot application. It can run on Cloud Foundry or your laptop, but it is more
-common to run the server in Cloud Foundry.
+The Spring Cloud Data Flow server deploys streams (long-lived applications) and tasks (short-lived applications) to Cloud Foundry.
```
[the remark this answers is not in the document]
Comment (tzolov) [reply spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc]: +1

File: spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc [hunk 3]
```diff
@@ -77,7 +70,8 @@ wget http://repo.spring.io/{version-type-lowercase}/org/springframework/cloud/sp
 wget http://repo.spring.io/{dataflow-version-type-lowercase}/org/springframework/cloud/spring-cloud-dataflow-shell/{dataflow-project-version}/spring-cloud-dataflow-shell-{dataflow-project-version}.jar
 ----
 ====
-. Optionally, download http://cloud.spring.io/spring-cloud-skipper/[Skipper] if you want the added features of upgrading and rolling back Streams since Data Flow delegates to Skipper for those features. The following example shows how to do so:
+. Download http://cloud.spring.io/spring-cloud-skipper/[Skipper] to which Data Flow delegate Streams lifecycle operations such as deployment, upgrading and rolling.
```
[the remark this answers is not in the document]
Comment (tzolov) [reply spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc]: +1

File: spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc [hunk 4]
```diff
@@ -423,12 +369,12 @@ dataflow-server              started           1/1         1G       1G     dataf
 ----
 ====
 
-The following example shows how to start the Data Flow Shell for the Data Flow server running in `skipper` mode:
+The following example shows how to start the Data Flow Shell for the Data Flow server:
```
[the remark this answers is not in the document]
Comment (tzolov) [reply spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc]: +1

File: spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc [hunk 5]
```diff
@@ -349,25 +311,25 @@ dataflow:>app import --uri http://bit.ly/Darwin-SR1-stream-applications-rabbit-m
...
+NOTE: Make sure the Skipper server is deployed and have configured the Data Flow server's `SPRING_CLOUD_SKIPPER_CLIENT_SERVER_URI` property to reference the Skipper server.
+
+The following example shows how to create and deploy a stream:
 
 ====
 [source]
 ----
-dataflow:> stream create --name httptest --definition "http | log" --deploy
+dataflow:> stream create --name httptest --definition "http | log"
+dataflow:> stream deploy --name httptest --platformName pws
```
[the remark this answers is not in the document]
Comment (tzolov) [reply spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc]: There is something called a default platform. But in the past we run in couple of issues when relying on the default platforms in non-local SCDF mode. 
Therefore the preferred approach is to set the platform explicitly. 
Still @markpollack and @sabbyanandan may have different opinion?

**Comment by cppwfs** [comment]
LGTM Squashed, Merged.

**Pull request closed without merging**